### DISCUSSION TITLE
How to disable the webserver?

### DISCUSSION THREAD

**Discussion by davideroffo** [serverpod/serverpod#650] [Q&A] [answered]
Is it possible to disable the webserver (default port 8082) and run only the standard server (default port 8080) and insights (default port 8081)?

**Comment by davideroffo** [comment, marked as answer]
I found that this answers my question: https://github.com/serverpod/serverpod/discussions/274#discussioncomment-3623318

**Reply by vlidholt** [reply]
Happy you found a solution. :)

**Comment by vlidholt** [comment]
Adding this for reference too: Just don't add any routes to the web server and it will not be started.
